db/migrate/20170315095936_update_persistent_volumes_parent_type.rb

```diff
@@ -0,0 +1,7 @@
+class UpdatePersistentVolumesParentType < ActiveRecord::Migration[5.0]
+  def change
+    PersistentVolume.all.each do |pv|
```

Comment:
Also, I believe this would auto-load `app/models/persistent_volume.rb` which is not allowed in migrations; you should define a tiny PersistentVolume class here.
See https://github.com/ManageIQ/manageiq/blob/master/db/migrate/20170207173837_set_service_display_and_retired_to_false.rb for an example.  That example also shows some STI-disabling magic which might or might not be good here, I have absolutely no idea...

Running `rake db:migrate` *without GOOD_MIGRATIONS_SKIP* should confirm it.

cc @lpichler Is it OK that Travis passed with such a migration?  Can we detect model-loading migrations in CI instead of detecting them when people pull master and run db:migrate?